BrandonBeltran/hackerrank/03/RansomNote.java

```diff
@@ -0,0 +1,63 @@
+package bootcamp;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Stream;
+import static java.util.stream.Collectors.toList;
+
+class Result {
+
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+        HashMap<String, Integer> magazineWords = new HashMap<>();
+
+        for(String word: magazine){
```

Comment:
Two foor loop almost identical... hmmmmmmmmm
Maybe another method that receives a list and an operation ? IDK
If you have another option, please let me know

Reply:
I moved things a little bit, please let me know how this solutions looks!